tests/unit/_memory_storage_client/test_memory_storage_client.py

```diff
@@ -224,13 +224,15 @@ async def test_not_implemented_method(tmp_path: Path) -> None:
 async def test_default_storage_path_used(monkeypatch: pytest.MonkeyPatch) -> None:
     # We expect the default value to be used
     monkeypatch.delenv('CRAWLEE_STORAGE_DIR', raising=False)
+    service_locator.set_configuration(Configuration())
```

Comment:
Is this necessary? Shouldn't the `service_locator` be empty at the start of the test?

Reply:
And couldn't we remove the configuration from `service_locator` and let the test trigger the "defaulting" on its own?